zschema/tests.py

```diff
@@ -114,12 +114,32 @@ def test_invalid(self):
         "type": "INTEGER",
         "name": "ip",
         "mode": "NULLABLE"
-    }
+    },
 ]
 
 
 class CompileAndValidationTests(unittest.TestCase):
 
+    def assertBigQuerySchemaEqual(self, a, b):
```

Comment:
I think you can probably replace this function with ```json.dumps(a, sort_keys=True) == json.dumps(b, sort_keys=True)``` This is likely safer than trying to reimplement.